src/components/PopoverMenu/popoverMenuPropTypes.js

```diff
@@ -54,6 +60,8 @@ const propTypes = {
 const defaultProps = {
     animationIn: 'fadeIn',
     animationOut: 'fadeOut',
+    animationInTiming: 300,
+    animationOutTiming: 300,
```

Comment:
We're only using the default value do we need this?

Reply:
@Uros787 I think we should remove all the references of `animationOutTiming` right?

Reply:
sure makes sense, i was leaving it there in case we were going to use it later.
removed